scheduler/plugin/host_test.go

```diff
@@ -0,0 +1,93 @@
+/*
+   Copyright 2023 The Kubernetes Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package wasm
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/tetratelabs/wazero/experimental/wazerotest"
+	"k8s.io/klog/v2"
+	k8stest "k8s.io/klog/v2/test"
+)
+
+func Test_k8sKlogLogFn(t *testing.T) {
```

Comment:
@sanposhiho these tests are new, using "wazerotest" which might be released in the next version of wazero. If you are cool `/lgtm` or otherwise lemme know. I intentionally kept them simple (vs using tables etc) for the first go.

cc @ncruces and @achille-roussel the first to use this style!

Reply:
Cool!